### ISSUE TITLE
Feature Request: Add example of adding handlers/commands to demo app

### ISSUE THREAD

**Issue by AshleyPoole** [monzo/response#58]
Could an example of how to add additional handlers/commands be added to the demo app so consumers of the new architecture (as a package) know how they can add their own items that aren't core to the Response package?

More of a question, as an side to this, would consumers be able to override certainly handlers/actions? For example, at the moment we have a custom channel name convention for incidents -- instead of an incrementing number as the default channel name, we switched to date time to work around the issue in a development environment where the channels already existed.

**Comment by milesbxf** [comment]
Yes, I really like this idea - better than documentation, as we can write tests for it and check that new releases haven't regressed anything.


> More of a question, as an side to this, would consumers be able to override certainly handlers/actions? For example, at the moment we have a custom channel name convention for incidents -- instead of an incrementing number as the default channel name, we switched to date time to work around the issue in a development environment where the channels already existed.

Definitely. We have this in the Monzo private fork, so we'll definitely need to port this over to the response package - I'm thinking of some kind of callback/hook that lets users plug in their own logic.

An even cooler feature I had an idea for was automatically generating a channel name from the incident name, prefixed by date - e.g. "External API latency high" could become `#inc-jul15-ext-api-lat` or something - we've found that there is a minute or two wasted at the beginning of incidents by having to rename the channel
